### DISCUSSION TITLE
Can we use and/or cite validation datasets in publications?

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#9241] [Q&A] [answered]
For example, can we select part of the Wheat validation dataset that uses cultivar X, create prediction-observed plots and use it in a paper along with an appropriate citation? If yes, what that citation must be?

**Comment by hol353** [comment]
Yes, it would be nice to be able to do this but currently it isn't possible. We have been pondering whether we change the way validation datasets are stored, perhaps making each experiment a self-contained directory, with it's own stats and a citation. Currently each model (e.g. wheat) has a directory that contains many experiments which makes model validations very large and a bit messy. We haven't moved beyond the 'dreaming what is possible' stage yet.

**Reply by BrianCollinss** [reply]
From the IP perspective, can one use one or more of the graphs from Wheat validation in a paper and cite APSIM-NextGen main paper?

**Reply by hol353** [reply, marked as answer]
I don't see any reason why not. Everything on GitHub is publically available and probably been published in various journals. Yes, cite the main APSIM paper.

**Reply by HamishBrownPFR** [reply]
From an IP perspective there is nothing stopping this.  However, I am not sure it would be good form to do so.  Alot of effort has gone into building the wheat model and validation set in apsim but there is not a general citation that describes the wheat model and validation.  It is still under active development and we have plans to publish as part of current projects.  Having certain graphs picked out of the validation set might cut across publication plans.
I would be keen to know more about what graphs are to be used and how.

**Reply by BrianCollinss** [reply]
@HamishBrownPFR Thanks for the comment Hamish. Currently there is no plan. But, from time to time we use APSIM (especially Wheat) with certain cultivars that have been used and tested in the validation dataset. So, my question was about such circumstances where we are asked to provide evidence of model performance and we need to provide that evidence without infringing any IP that we don't own.

**Reply by HamishBrownPFR** [reply]
@BrianCollinss.  I don't think it would cause any problems if you were just wanting to slice a few cultivars out of the validation set to make a graph supporting their use in a simulation exercise.

**Reply by BrianCollinss** [reply]
Obviously if there is a memo there to recommend citing a certain paper, it must be followed.
